dali/operators/geometry/affine_transforms/translate_transform.cc

```diff
@@ -12,14 +12,24 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "dali/operators/geometry/transform_base_op.h"
+#include "dali/operators/geometry/affine_transforms/transform_base_op.h"
 #include "dali/pipeline/data/views.h"
 
 namespace dali {
 
 DALI_SCHEMA(TranslateTransform)
-  .DocStr(R"(TranslateTransform)")
-  .AddArg("offset", "Translation vector", DALI_FLOAT_VEC, true)
+  .DocStr(R"(Produces a translation affine transform.
+
+If another transform is passed as an input, the operator will combine the two transforms by applying
+translation to the transform provided.
+
+.. note::
+    The output of this operator can be fed directly to ``MT`` argument of ``CoordTransform`` operator.
```

Comment:
Can we use it in Warp operator for images as well?

Reply:
As far as I know, WarpAffine operator expects the inverse of the transform, so it can't be directly fed into it until WarpAffine implements the flag to accept inverse transforms. @mzient Correct?

Reply:
For translation, we'd just need to use the negated translation vector.